AUTHORS

```diff
@@ -50,14 +64,18 @@ Martin Zachrison
 matija.mazi@gmail.com
 Matt Bogosian
 Matt Corallo
+Matthew Shaylor
 Michael Bell
 Michael Bumann
 Mike Hearn
 Mike Rosseel
 Miron Cuperman
+Miron Cuperman (devrandom)
 monk
 Mora Zyx
 mruddy
+Nicola
+Nicola Atzei
```

Comment:
If you interactively rebase your branch on master it's easy to take out or edit the commit. But sure, I can also leave it out when merging.